### ISSUE TITLE
[CAIA Future Support][A11y Maintenance Audit]: Pension 527EZ Form Updates - Benefits and Claims

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#67293]
`Update 10/11/2023: Per DSVA Slack with Kate and Laura, this audit is not needed immediately`

## CAIA Maintenance Audit
**There has never been a maintenance audit on this form.** 
- The project team is requesting one be completed, [as per the kickoff meeting on 10/3/2023](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/meeting-notes/2023-10-03-Kickoff-Pension-Reactivation.md) 
- The following are the supporting artifacts provided in the intake ticket #66578 c/o @steele-lm 
    - This ticket is connected via the a11y task list found in the intake ticket

### Task List
- **There is also a`task list`, breaking out the audit tasks.** @ohgreatkate @coforma-terry 
    - This was done, should CAIA team members need to divvy this up. 
    - Names can be added in the placeholder field. 
 
### Supporting artifacts
- **CAIA Kickoff Meeting 10/3/2023 Deliverable:** [Meeting Notes](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/meeting-notes/2023-10-03-Kickoff-Pension-Reactivation.md)
- Product outline: [Pension Online Form Reactivation](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/pension/527EZ-reactivation-initiative-brief.md)
- Any epics/issues that may be helpful: 
    - https://github.com/department-of-veterans-affairs/va.gov-team/issues/63349
    - https://github.com/department-of-veterans-affairs/va.gov-team/issues/65250 
    - https://github.com/department-of-veterans-affairs/va.gov-team/issues/67343
- Link to testing accounts (please do not paste them in this ticket): [527EZ test accounts](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/pension/form-deactivation/527EZ-staging-test-accounts-accessible.md)

### Definition of done

1. Review and acknowledge all issues added to this maintenance audit.
2. Fix and/or document decisions made for all issues.
3. A11y specialist will review documented decisions or validating fixes for all tickets.
4. A11y specialist will update the product page on the VA a11y dashboard.
5. A11y specialist will close this ticket.

### Auditor(s)

Kate Murphy

### Automated

- [ ] Run axe checks on every page or unique state (required)
- [ ] Run WAVE on every page if time allows. WAVE is more visual than axe or axe-coconut, but offers some excellent ways to identify nested headings and HTML5 landmark tags.

### Manual

- [ ] Color contrast checks (recommend using whocanuse.com for beyond compliance reports)
- [ ] Color blindness checks
- [ ] Zoom layouts to 400% and inspect them for readability. If layouts break at 400%, I will start reducing them until they become stable, and log the zoom ratio when things started breaking.
- [ ] Keyboard navigation

### Screen Reader

- [ ] Windows, Chrome, JAWS
- [ ] iPhone, iOS Safari, VoiceOver

### Other tests

1. for example, Dragon Naturally Speaking on Chrome on Windows
2.
3.
...

### Audit sheet link

_Needs created_

### Production link

_Needs provided_

### Test Users
[527EZ test accounts](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/pension/form-deactivation/527EZ-staging-test-accounts-accessible.md)
1.
2.
3.
...

```[tasklist]
### Future Item: a11y Maintenance Audit Segments
- [ ] [a11y Audit] Segment: Automated Testing (CAIA Team Member Name) - Pension 527EZ Form
- [ ] [a11y Audit] Segment:Manual Testing (CAIA Team Member Name) - Pension 527EZ Form
- [ ] [a11y Audit] Segment:Screen Reader Testing (CAIA Team Member Name) - Pension 527EZ Form
- [ ] [a11y Audit] Segment: Other Testing (CAIA Team Member Name) - Pension 527EZ Form
```
```[tasklist]
### a11y Ticket Prep
- [x] Create Audit Ticket
- [x] Ticket refinement: labels, assignees, project and project fields
- [x] Tie this ticket to intake ticket
- [x] Add task list for audit subtasks
- [x] Add links to supporting artifacts
- [x] Include kick-off meeting notes to ticket
- [x] Add design intent zoom link and a11y findings
- [x] Flag as future support based on Kate and Laura's DSVA Slack
- [x] Flagging as low priority based on it being a future item
```

**Comment by vbellissimo** [comment]
@coforma-terry @sara-amanda @katherine-fung Is this ticket okay to be closed now that we're launched? It may also have duplicates that now exist for bringing back entry points to the Pension form and the URL updates we're planning once we're back to 100% of traffic.

**Comment by sara-amanda** [comment]
> Is this ticket okay to be closed now that we're launched? It may also have duplicates that now exist for bringing back entry points to the Pension form and the URL updates we're planning once we're back to 100% of traffic.


Good morning, @vbellissimo ! 

- We can always do a post-launch audit? 
    - When will your redirects be in place? 
- Can you provide the link to run the audit on, if you would like to do this, prior to us closing this ticket out? 

cc: @EvanAtCoforma @coforma-terry @coforma-jamie
